sdk/containerregistry/container-registry/README.md

```diff
@@ -0,0 +1,80 @@
+# Azure Container Registry client library for JavaScript
+
+Azure [Container Registry](https://azure.microsoft.com/services/container-registry/) is a managed, private Docker registry service based on the open-source Docker Registry 2.0. Create and maintain Azure container registries to store and manage your private Docker container images and related artifacts.
+
+Use Azure container registries with your existing container development and deployment pipelines, or use Azure Container Registry Tasks to build container images in Azure. Build on demand, or fully automate builds with triggers such as source code commits and base image updates.
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 8.x or higher
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An [Azure Container Registry resource][acr_resource]
+
+Usually you'd put a shell command for provisioning the necessary Azure services here.
+
+### Install the `@azure/container-registry` package
+
+Install the Container Registry client library for JavaScript with `npm`:
+
+```bash
+npm install @azure/container-registry
+```
+
+### Browser support
+
+#### JavaScript Bundle
+
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
+
+### Further examples
+
+Top-level examples usually include things like creating and authenticating the main Client. If your service supports multiple means of authenticating (e.g. key-based and Azure Active Directory) you can give a separate example of each.
+
+## Key concepts
+
+### RepositoryClient
+
+`RepositoryClient` provides operations to interact with a repository.
```

Comment:
I only see `ContainerRegistryClient` in the API file, is `RepositoryClient` a future construct?

Reply:
I renamed the client but missed README.

Reply:
Fixed